Anatolie/src/main/java/Text.java

```diff
@@ -0,0 +1,9 @@
+public class Text {
+        public static void main(String[] args) {
+            String lower_string = "HeLLo";
+            String upper_String = "heLLO";
+            if (lower_string != upper_String) {
```

Comment:
Its not save to use == operator to compare strings
see also https://www.programiz.com/java-programming/examples/compare-strings#:~:text=You%20should%20not%20use%20%3D%3D,and%20not%20the%20object%20itself.

Reply:
Done